public/locales/fr/email.json

```diff
@@ -1,6 +1,6 @@
 {
-  "header": "Obtenir mon numéro de dossier (FEDS)",
-  "description": "Veuillez saisir les champs obligatoires suivants, tels qu'ils figurent sur votre formulaire de demande de passeport, pour que votre numéro de dossier (FEDS) soit envoyé par courriel à l'adresse courriel au dossier.",
+  "header": "Obtenir mon numéro de dossier",
+  "description": "Remplissez les champs ci-dessous pour obtenir votre numéro de dossier. Assurez-vous que vos informations correspondent à votre formulaire de demande de passeport. Nous enverrons ensuite le numéro de dossier par e-mail à l'adresse e-mail enregistrée.",
```

Comment:
Change `e-mail` to `courriel` in this file or maybe in other French locales as well. I didn't look at the other files.